cms/djangoapps/contentstore/views/course.py

```diff
@@ -824,6 +832,15 @@ def _create_or_rerun_course(request):
         else:
             try:
                 new_course = create_new_course(request.user, org, course, run, fields)
+
+                # [COLARAZ_CUSTOM]
+                # Link courses with all selected secondary organiztaions
+                secondary_organizations = request.json.get('secondary_orgs')
```

Comment:
nit. just a suggestion to avoid nesting.
```suggestion
                secondary_organizations = request.json.get('secondary_orgs', [])
```

This way we can avoid the if condition check..
You can ignore this if you want to.

Reply:
The default value is only useful when there is a missing key. In our case key is there but with value "None" so default value will not work here.